.github/workflows/regression.yml

```diff
@@ -754,8 +756,20 @@ jobs:
     env:
       FAILED: ${{ contains(needs.*.result, 'failure') }}
     steps:
+      # Check if the actor is Dependabot and skip if true, unless it's a review
```

Comment:
Rather than looking for `dependabot[bot]` could we instead look for a label like `ci/skip-regression`? I think that would have the advantage of letting us add the tag to PRs that we want to have skip the regression tests (e.g. docs changes).

You can change the labels that dependabot updates by default with: https://github.com/vectordotdev/vector/blob/441bf23e590fcc4f0043657beca25318f8c16cf5/.github/dependabot.yml#L8-L10